osu.Game.Tests/Visual/UserInterface/TestSceneOsuHoverContainer.cs

```diff
@@ -55,8 +55,6 @@ public void TestIsHoveredHasProperValue(bool isEnabled)
 
             moveOut();
             checkNotHovered();
-
-            ReturnUserInput();
```

Comment:
Could you please explain me why not return the user input? Otherwise if no return user inputs are among test cases, from test GUI one can't move the cursor inside. Or is `ReturnUserInput()` called somewhere automatically in a test base?

Reply:
See https://github.com/ppy/osu/pull/4945